src/main/java/com/jalasoft/jfc/controller/PdfConverterController.java

```diff
@@ -128,7 +127,9 @@
             fileResponse.setStatus(MessageResponse.SUCCESS200.getMessageResponse());
 
             return new ResponseEntity<>(fileResponse, HttpStatus.OK);
-        } catch (ConvertException | Md5Exception ex) {
+        } catch (ConvertException ex) {
+            return new ResponseEntity<>(fileResponse, HttpStatus.OK);
```

Comment:
please fix this because there aren't errorResponse as the other catch

Reply:
put this catch (ConvertException | Md5Exception ex)
with its respective errorResponse.

} catch (ConvertException | Md5Exception ex) {
            errorResponse.setName(pdfParam.getOutputName());
            errorResponse.setStatus(MessageResponse.ERROR406.getMessageResponse());
            errorResponse.setError(ex.toString());
            return new ResponseEntity<>(errorResponse, HttpStatus.NOT_ACCEPTABLE);